Composer/packages/client/src/components/GetStarted/GetStarted.tsx

```diff
@@ -0,0 +1,211 @@
+
+  const linkToPackageManager = `/bot/${projectId}/plugin/package-manager/package-manager`;
+  const linkToConnections = `/bot/${projectId}/botProjectsSettings/#connections`;
+  const linkToProvision = `/bot/${projectId}/botProjectsSettings/#addNewPublishProfile`;
+  const linkToPublish = `/bot/${projectId}/publish`;
+  const linkToLGEditor = `/bot/${projectId}/language-generation`;
+  const linkToLUEditor = `/bot/${projectId}/language-understanding`;
+  const linkToDelete = `/bot/${projectId}/botProjectsSettings/#deleteBot`;
+  const linkToAdaptiveExpressions =
+    'https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-concept-adaptive-expressions?view=azure-bot-service-4.0&tabs=arithmetic';
+  const linkToGetStarted = 'https://docs.microsoft.com/en-us/composer/introduction';
+  const linkToCreateFirstBot = 'https://docs.microsoft.com/en-us/composer/quickstart-create-bot';
+  const linkToTutorials = 'https://docs.microsoft.com/en-us/composer/tutorial/tutorial-introduction';
+  const linkToLGFileFormat =
+    'https://docs.microsoft.com/en-us/azure/bot-service/file-format/bot-builder-lg-file-format?view=azure-bot-service-4.0';
+  const linkToLUFileFormat =
+    'https://docs.microsoft.com/en-us/azure/bot-service/file-format/bot-builder-lu-file-format?view=azure-bot-service-4.0';
+  const linkToPreBuiltExpressions =
+    'https://docs.microsoft.com/en-us/azure/bot-service/adaptive-expressions/adaptive-expressions-prebuilt-functions?view=azure-bot-service-4.0';
+
+  const items: IToolbarItem[] = [
+    {
+      text: formatMessage('Delete bot'),
+      type: 'action',
+      buttonProps: {
+        iconProps: { iconName: 'Trash' },
+        onClick: () => buttonClick(linkToDelete),
+        styles: buttonStyles,
+      },
+      align: 'left',
+    },
+    {
+      text: formatMessage('Add a package'),
+      type: 'action',
+      buttonProps: {
+        iconProps: { iconName: 'Package' },
+        onClick: () => buttonClick(linkToPackageManager),
+        styles: buttonStyles,
+      },
+      align: 'left',
+    },
+    {
+      text: 'Edit LG',
+      type: 'action',
+      buttonProps: {
+        iconProps: { iconName: 'Robot' },
+        onClick: () => buttonClick(linkToLGEditor),
+        styles: buttonStyles,
+      },
+      align: 'left',
+    },
+    {
+      text: 'Edit LU',
+      type: 'action',
+      buttonProps: {
+        iconProps: { iconName: 'People' },
+        onClick: () => buttonClick(linkToLUEditor),
+        styles: buttonStyles,
+      },
+      align: 'left',
+    },
+    {
+      text: 'Manage connections',
+      type: 'action',
+      buttonProps: {
+        iconProps: { iconName: 'PlugConnected' },
+        onClick: () => buttonClick(linkToConnections),
+        styles: buttonStyles,
+      },
+      align: 'left',
+    },
+  ];
+
+  const linkClick = (event) => {
+    // TODO: track telemetry!
+  };
+
+  const buttonClick = (link) => {
+    // TODO: track telemetry
```

Comment:
is this coming in with this PR?

Reply:
was going to do a telemetry follow-up for this and a few other spots.

Reply:
done